docs/docs/concepts/plugins.mdx

```diff
@@ -517,19 +535,35 @@ export TAP_SMOKE_TEST__SELECT='["animals\\.data.id","animals\\.data.verified"]'
 - [`meltano el`](/reference/command-line-interface#el) CLI options: `--select` and `--exclude`
 - Default: `[]`
 
-An extractor's `select_filter` [extra](/guide/configuration#plugin-extras) holds an array of [entity selection](#select-extra) filter rules
+An extractor's `select_filter` [extra](/guide/configuration#plugin-extras) holds an array of stream-level filter rules
 that are applied to the extractor's [discovered](/guide/integration#extractor-catalog-generation) or [provided](#catalog-extra) catalog file
 when the extractor is run using [`meltano run`](/reference/command-line-interface#run), [`meltano invoke`](/reference/command-line-interface#invoke), or [`meltano el`](/reference/command-line-interface#el),
-after [schema](#schema-extra), [selection](#select-extra), and [metadata](#metadata-extra) rules are applied.
+**after** [schema](#schema-extra), [selection](#select-extra), and [metadata](#metadata-extra) rules are applied.
+
+The `select_filter` extra operates at the **stream level** and determines **which entire streams** should be included or excluded from extraction. It preserves any property-level selections you've made with the [`select` extra](#select-extra).
 
-It can be used to only extract records for specific matching entities, or to extract records for all entities _except_ for those specified, by letting you apply filters on top of configured [entity selection rules](#select-extra).
+## Key Differences: `select` vs `select_filter`
 
-Selection filter rules use entity identifiers that correspond to Singer stream `tap_stream_id` values. Rules indicating that an entity should be excluded are prefixed with an exclamation mark (`!`). [Unix shell-style wildcards](<https://en.wikipedia.org/wiki/Glob_(programming)#Syntax>) can be used in entity identifiers to match multiple entities at once.
+| `select` | `select_filter` |
+|----------|----------------|
+| **Property-level**: Controls which properties within streams are included | **Stream-level**: Controls which entire streams are included/excluded |
+| **Configuration-time**: Set during project configuration | **Runtime**: Typically set via CLI options (`--select`, `--exclude`) |
```

Comment:
? Runtime I would say would typically be set via ENV vars, not CLI options as [run ](https://deploy-preview-9434--meltano.netlify.app/reference/command-line-interface/#parameters-3)doesn't have a parameter for select like those listed 

Very confusing comparison to me